install_config/oab_broker_configuration.adoc

```diff
@@ -90,6 +90,15 @@ secrets: <6>
 <5> See xref:oab-config-broker[Broker Configuration] for details.
 <6> See xref:oab-config-secrets[Secrets Configuration] for details.
 
+[[install-config-oab-running-docker]]
+== Running Docker on each node
```

Comment:
For 3.11, headings are in title case. Also, might need to revisit this title depending on my comment below about the intro sentence.

Reply:
Title changed to Authenticating on Red Hat Connect Partner Registry